iosApp/iosApp/Artist/ArtistView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ArtistView.swift
+//  iosApp
+//
+//  Created by Marty Ulrich on 1/10/22.
+//  Copyright © 2022 orgName. All rights reserved.
```

Comment:
Do we need the copyright header?

Reply:
Nah it’s just defualt, didn’t even notice. Although since the repo’s public maybe we do? 🤷‍♂️